syncd/scripts/physyncd_init_common.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
```

Comment:
can we reuse syncd_init_common.sh?

Reply:
Probably, so, an approach would be to pass an argument to the common script to indicate if we were launched by syncd_start.sh or physyncd_start.sh, and then put if statements around the code that does not make sense for a phy (for example, we would not be checking platform type and calling functions like config_syncd_vs, we would instead look at the argument we passed and call config_physyncd). Would that be acceptable?

Reply:
Addressed, please review

Reply:
@lguohan Addressed, thanks for the catch.